EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/getting_started/installation.md

**Prerequisite**:

> Make sure that you have setup proper auth parameters for the updates.ez.no update server. You will receive the login parameters from the customer portal at ez.no:

Reviewer: Per our terminology "setup" is a noun, while "set up" is a verb. I'd also spell out "authentication" to avoid jargon. If authentication parameters and login parameters are the same, I would choose one option and apply it consistently. Also - I would remove "will" and simply inform the user that "You receive the ... parmeters..."

> Make sure that you have setup proper auth parameters for the updates.ez.no update server. You will receive the login parameters from the customer portal at ez.no:

Reviewer 2: ```suggestion
Make sure to configure proper authentication parameters for the updates.ez.no update server. You will receive the login parameters from the customer portal at <https://www.ibexa.co/>:
```
